Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/Networking/Forge/Networking/FacepunchNetworkingPlayer.cs

```diff
@@ -0,0 +1,17 @@
+﻿#if FACEPUNCH_STEAMWORKS
+using Steamworks;
+
+namespace BeardedManStudios.Forge.Networking
+{
+	public class FacepunchNetworkingPlayer : NetworkingPlayer
+	{
+		public UDPPacketManager PacketManager { get; private set; }
+
+		public FacepunchNetworkingPlayer(uint networkId, SteamId steamId, bool isHost, NetWorker networker) : base(networkId, "", isHost, null, networker)
```

Comment:
Is `SteamId` defined or is there a `using` directive missing? Also too bad the Facepunch implementation defined their own type as besides this change the whole file is exactly the same as the SteamNetworkingPlayer. Maybe we could roll the two together and have two overloaded constructors?

Reply:
I was thinking of something like this. What do you think?

```cs
#if STEAMWORKS || FACEPUNCH_STEAMWORKS
using Steamworks;

namespace BeardedManStudios.Forge.Networking
{
    public class SteamNetworkingPlayer : NetworkingPlayer
    {
        public UDPPacketManager PacketManager { get; private set; }

	    #if STEAMWORKS
        public SteamNetworkingPlayer(uint networkId, CSteamID steamId, bool isHost, NetWorker networker) : base(networkId, "", isHost, null, networker)
        #else
	    public SteamNetworkingPlayer(uint networkId, SteamId steamId, bool isHost, NetWorker networker) : base(networkId, "", isHost, null, networker)
	    #endif
		{
            PacketManager = new UDPPacketManager();
            SteamID = steamId;
        }
    }
}
#endif
```

Reply:
using dir on line 2 looks fine to me, is there a mistake here? Agree it would be slicker to combine with the two and write a second constructor.